src/outline/manager.test.ts

```diff
@@ -0,0 +1,99 @@
+/* eslint-env jest */
+import OutlineManager from './manager'
+
+import { lit, NamedNode, sym } from 'rdflib'
+import { findByText, getByText } from '@testing-library/dom'
```

Comment:
I wasn't sure about using testing-library?  I think we may have tried to stay with jsdom.  I can't remember why exactly.  This is something to look at though, because potentially if we can use the testing-library/dom there may be a chance I could use it in solid-ui for draganddrop because that is not supported with jsdom.

Reply:
jsdom is not a testing library, just a node implementation of dom. @testing-library/dom is imho the perfect fit to test our functionality so I do not see why we should not use it.